pkg/flash/op/op.go

```diff
@@ -6,29 +6,72 @@
 // the beginning of a SPI transaction.
 package op
 
+import "fmt"
+
+type OpCode byte
+
 const (
 	// PageProgram programs a page on the flash chip.
-	PageProgram byte = 0x02
+	PageProgram OpCode = 0x02
 	// Read reads from the flash chip.
-	Read byte = 0x03
+	Read OpCode = 0x03
 	// WriteDisable disables writing.
-	WriteDisable byte = 0x04
+	WriteDisable OpCode = 0x04
 	// ReadStatus reads the status register.
-	ReadStatus byte = 0x05
+	ReadStatus OpCode = 0x05
 	// WriteEnable enables writing.
-	WriteEnable byte = 0x06
+	WriteEnable OpCode = 0x06
 	// SectorErase erases a sector to the value 0xff.
-	SectorErase byte = 0x20
+	SectorErase OpCode = 0x20
 	// ReadSFDP reads from the SFDP.
-	ReadSFDP byte = 0x5a
+	ReadSFDP OpCode = 0x5a
 	// ReadID reads the JEDEC ID.
-	ReadJEDECID byte = 0x9f
+	ReadJEDECID OpCode = 0x9f
 	// PRD/RES
-	PRDRES = 0xab
-	// Enter4BA enters 4-byte addressing mode.
-	Enter4BA byte = 0xb7
+	PRDRES OpCode = 0xab
+	// AAI is auto address increment
+	AAI OpCode = 0xad
+	// Enter4BA enters 4-OpCode addressing mode.
+	Enter4BA OpCode = 0xb7
 	// BlockErase erases a block to the value 0xff.
-	BlockErase byte = 0xd8
-	// Exit4BA exits 4-byte addressing mode.
-	Exit4BA byte = 0xe9
+	BlockErase OpCode = 0xd8
+	// Exit4BA exits 4-OpCode addressing mode.
+	Exit4BA OpCode = 0xe9
 )
+
+func (o OpCode) String() string {
+	switch o {
```

Comment:
You can replace this with a

```
if val, ok := map[OpCode]string{
 PageProgram: "PageProgram",
 ...
}; ok {
  return val
}
return fmt.Sprintf("Unknown(%02x)", byte(o))
```

Then it's easy to appease the codecov with a single unit test.

Reply:
I just did the unit test. Having a map like that lying around requires an initialization and the footprint is non-zero for something we almost never need. I'm sensitive to this kind of thing now in a world of tinygo.